### ISSUE TITLE
Update Documentation of Symfony PHPUnit Bridge for Reusable Bundles

### ISSUE THREAD

**Issue by ReservedDeveloper** [symfony/symfony-docs#5901]
This is a follow up in relation to symfony/symfony-docs#5059. Presently, there does not seem to be a clear way to utilize the PHPUnit Bridge library in conjunction with a reusable bundle configuration. 

I installed this library for a current project and started receiving deprecation notices from a self-maintained dependency. After installing it on the dependency, I'm not getting any deprecation notices. I'm not sure if its a matter of misconfiguration in the dependency project, or that the bridge package isn't set up to handle reusable bundles.

It would be helpful if the documentation were updated to cover that use case, or updated to reflect that not being a valid approach.

**Comment by ReservedDeveloper** [comment]
See here for my [SO post detailing my setup](http://stackoverflow.com/q/33157336/4375304) a bit more (and a bounty, if someone can help sort this out!).

Also, ping to @nicolas-grekas @xabbuh and @WouterJ, who I suspect will be best equipped for this task.

**Comment by nicolas-grekas** [comment]
IMHO, the doc/README is fine, it tells what it should tell.
If you get more notices on your bundle than in your dependency, it means you are missing some tests cases in your dependencies and that your bundle runs throught code paths that are not taken by your dependencie's test suite.

**Comment by ReservedDeveloper** [comment]
@nicolas-grekas that is true. In my case, since this is a reusable bundle, none of the service configuration code is hit directly, since that's the responsibility of the dependent project to load up and parse.

I think ultimately, I need to take an approach [similar to the FOS Rest bundle](https://github.com/FriendsOfSymfony/FOSRestBundle/blob/4d103b/Tests/DependencyInjection/FOSRestExtensionTest.php) in testing the Extension class, which should resolve the unparsed code issue. Perhaps this approach would be worth mentioning in the Reusable Bundles best practices section. Good for testing all lot of those annotation/configuration type issues.
